src/frontend/components/Filter/Filter.js

```diff
@@ -52,10 +64,220 @@ function Filter() {
+            Love / Peace
+          </label>
+        </div>
+      </Accordion>
 
-      {data.map((item, i) => (
-        <Accordion key={i} data={item} />
-      ))}
+      <Accordion title="User Ratings">
+        <div className="accordion-input">
+          <label htmlFor="fourStars">
+            <input
+              type="radio"
+              id="fourStars"
+              name="ratings"
+              className="filter-input"
+              value="fourStar"
+              checked={state.userRatings === 4}
+              onChange={() =>
+                dispatch({
+                  type: BY_USER_RATINGS,
+                  payload: 4,
+                })
+              }
+            />
+            4 Stars & above
+          </label>
+        </div>
+        <div className="accordion-input">
+          <label htmlFor="threeStars">
+            <input
+              type="radio"
+              id="threeStars"
+              name="ratings"
+              className="filter-input"
+              checked={state.userRatings === 3}
+              onChange={() =>
+                dispatch({
+                  type: BY_USER_RATINGS,
+                  payload: 3,
+                })
+              }
+            />
+            3 Stars & above
+          </label>
+        </div>
+        <div className="accordion-input">
+          <label htmlFor="twoStars">
+            <input
+              type="radio"
+              id="twoStars"
+              name="ratings"
+              className="filter-input"
+              checked={state.userRatings === 2}
+              onChange={() =>
+                dispatch({
+                  type: BY_USER_RATINGS,
+                  payload: 2,
+                })
+              }
+            />
+            2 Stars & above
+          </label>
+        </div>
+      </Accordion>
+      <Accordion title="Sort By">
+        <div className="accordion-input">
+          <label htmlFor="logToHigh">
+            <input
+              type="radio"
+              id="logToHigh"
+              name="lowhigh"
+              className="filter-input"
+              checked={state.sortBy === "LOW_TO_HIGH"}
+              onChange={() =>
+                dispatch({
+                  type: SORT_BY,
+                  payload: "LOW_TO_HIGH",
+                })
+              }
+            />
+            Low to High
+          </label>
+        </div>
+        <div className="accordion-input">
+          <label htmlFor="highToLow">
+            <input
+              type="radio"
+              id="highToLow"
+              name="lowhigh"
+              className="filter-input"
+              checked={state.sortBy === "HIGH_TO_LOW"}
+              onChange={() =>
+                dispatch({
+                  type: SORT_BY,
+                  payload: "HIGH_TO_LOW",
+                })
+              }
+            />
+            High to Low
+          </label>
+        </div>
+      </Accordion>
     </section>
   );
 }
```

Comment:
This file is too huge. Maybe you could divide it into some components.

Reply:
Yup I'll Thank you.